src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,36 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        System.out.println();
+        String[] strDateFrom = dateFrom.split("\\.");
```

Comment:
[Any magic numbers (symbols and literals)  should be constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#any-magic-numbers-should-be-constants)